js/main.js

```diff
@@ -0,0 +1,45 @@
+let books = [
+  { title: 'Lorem', author: 'someone' },
+  { title: 'Lorem2', author: 'someone' },
+  { title: 'Lorem3', author: 'someone' },
+];
+
+function updateList() {
+  document.getElementById('books-list').innerHTML = books.map(
```

Comment:
- [ ] OPTIONAL: I recommend you to use ES6 syntax arrow function: 
`const updateList = ()  => { }`